backend/scripts/add_unique_ids/extra/map_ids.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+from csv import DictReader
+from pathlib import Path
+
+
+def main(cclw_newer_path: Path, exported_document_table: Path):
+    current_name_urls = {}
+    with open(exported_document_table) as doct:
+        csv_reader = DictReader(doct)
+        for r in csv_reader:
+            name = r["name"].strip()
+            url = r["source_url"].strip()
+            if (name, url) in current_name_urls:
+                print(name)
+            current_name_urls[(name, url)] = r["id"]
+
+    doc_id_to_import = {}
+    with open(cclw_newer_path) as cclw_newer:
+        csv_reader = DictReader(cclw_newer)
+        for r in csv_reader:
+            name = r["Title"].strip()
+            url = r["Documents"].split("|")[0].strip().replace("http://", "https://")
```

Comment:
why do we do this?

Reply:
because the CCLW input file is pure bonkers 🤣